Update owin-middleware-in-the-iis-integrated-pipeline.md (#780)
dotnet/AspNetDocs: aspnet/aspnet/overview/owin-and-katana/owin-middleware-in-the-iis-integrated-pipeline.md

## How OWIN Middleware Executes in the IIS Integrated Pipeline

> For OWIN console applications, the application pipeline built using the [startup configuration](owin-startup-class-detection.md) is set by the order the components are added using the `IAppBuilder.Use` method. That is, the OWIN pipeline in the [Katana](an-overview-of-project-katana.md) runtime will process OMCs in the order they were registered using `IAppBuilder.Use`. In the IIS integrated pipeline the request pipeline consists of [HttpModules](https://msdn.microsoft.com/library/ms178468(v=vs.85).aspx) subscribed to a pre-defined set of the pipeline events such as [BeginRequest](https://msdn.microsoft.com/library/system.web.httpapplication.beginrequest.aspx), [AuthenticateRequest](https://msdn.microsoft.com/library/system.web.httpapplication.authenticaterequest.aspx), [AuthorizeRequest](https://msdn.microsoft.com/library/system.web.httpapplication.authorizerequest.aspx), etc.  You may interested in how does the [Microsoft.Owin.Host.SystemWeb](https://www.nuget.org/packages/Microsoft.Owin.Host.SystemWeb) nuget package register the [`OwinHttpModule`](https://github.com/aspnet/AspNetKatana/blob/main/src/Microsoft.Owin.Host.SystemWeb/OwinHttpModule.cs) , normally, `HttpModule` is registed in IIS  via `Web.config` file, but `Microsoft.Owin.Host.SystemWeb` use a IIS feature called [`PreApplicationStartMethodAttribute`](https://docs.microsoft.com/en-us/dotnet/api/system.web.preapplicationstartmethodattribute?view=netframework-4.8) and [`HttpApplication.RegisterModule(Type)`](https://docs.microsoft.com/en-us/dotnet/api/system.web.httpapplication.registermodule?view=netframework-4.8) to dynamicly register the `OwinHttpModule` to IIS pipeline.

Reviewer: ```suggestion
For OWIN console applications, the application pipeline built using the [startup configuration](owin-startup-class-detection.md) is set by the order the components are added using the `IAppBuilder.Use` method. That is, the OWIN pipeline in the [Katana](an-overview-of-project-katana.md) runtime processes OMCs in the order they were registered using `IAppBuilder.Use`. In the IIS integrated pipeline the request pipeline consists of [HttpModules](https://msdn.microsoft.com/library/ms178468(v=vs.85).aspx) subscribed to a pre-defined set of the pipeline events such as [BeginRequest](https://msdn.microsoft.com/library/system.web.httpapplication.beginrequest.aspx), [AuthenticateRequest](https://msdn.microsoft.com/library/system.web.httpapplication.authenticaterequest.aspx), [AuthorizeRequest](https://msdn.microsoft.com/library/system.web.httpapplication.authorizerequest.aspx), etc.  Notice that the [Microsoft.Owin.Host.SystemWeb](https://www.nuget.org/packages/Microsoft.Owin.Host.SystemWeb) nuget package registers the [`OwinHttpModule`](https://github.com/aspnet/AspNetKatana/blob/main/src/Microsoft.Owin.Host.SystemWeb/OwinHttpModule.cs). Typically, `HttpModule` is registered in IIS  via `Web.config` file, but `Microsoft.Owin.Host.SystemWeb` use a IIS feature called [`PreApplicationStartMethodAttribute`](/dotnet/api/system.web.preapplicationstartmethodattribute) and [`HttpApplication.RegisterModule(Type)`](/dotnet/api/system.web.httpapplication.registermodule) to dynamically register the `OwinHttpModule` to IIS pipeline.
```
